lang/src/lib.rs

```diff
@@ -45,6 +45,7 @@ pub mod event;
 pub mod idl;
 pub mod system_program;
 mod vec;
+pub mod address_lookup_table_program;
```

Comment:
Given this statement has a similar functionality as the `mod bpf_upgradeable_state;` statement, I think it makes sense to keep their visibility the same.

```suggestion
mod address_lookup_table_program;
```